include/itkVkHalfHermitianToRealInverseFFTImageFilter.hxx

```diff
@@ -0,0 +1,123 @@
+/*=========================================================================
+ *
+ *  Copyright NumFOCUS
+ *
+ *  Licensed under the Apache License, Version 2.0 (the "License");
+ *  you may not use this file except in compliance with the License.
+ *  You may obtain a copy of the License at
+ *
+ *         http://www.apache.org/licenses/LICENSE-2.0.txt
+ *
+ *  Unless required by applicable law or agreed to in writing, software
+ *  distributed under the License is distributed on an "AS IS" BASIS,
+ *  WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ *  See the License for the specific language governing permissions and
+ *  limitations under the License.
+ *
+ *=========================================================================*/
+#ifndef itkVkHalfHermitianToRealInverseFFTImageFilter_hxx
+#define itkVkHalfHermitianToRealInverseFFTImageFilter_hxx
+
+#include "itkHalfToFullHermitianImageFilter.h"
+#include "itkVkHalfHermitianToRealInverseFFTImageFilter.h"
+#include "itkIndent.h"
+#include "itkMetaDataObject.h"
+#include "itkProgressReporter.h"
+#include "itkMultiThreaderBase.h"
+
+#include <iostream>
+
+namespace itk
+{
+
+template <typename TInputImage>
+VkHalfHermitianToRealInverseFFTImageFilter<TInputImage>::VkHalfHermitianToRealInverseFFTImageFilter()
+{
+  this->DynamicMultiThreadingOn();
```

Comment:
Calling dynamic threading on or off is not necessary if the filter is not multithreaded. That would mean the default is used. Removing this would make the code simpler and less confusing to people less knowledgeable about ITK insides.

Reply:
Thank you.  Will do.